pkg/gofr/gofr_test.go

```diff
@@ -66,28 +66,28 @@ func TestGofr_ServerRoutes(t *testing.T) {
 
 	g := New()
 
-	g.GET("/hello", func(c *Context) (interface{}, error) {
+	g.GET("/hello", func(_ *Context) (interface{}, error) {
 		return helloWorld, nil
 	})
 
 	// using add() func
-	g.add(http.MethodGet, "/hello2", func(c *Context) (interface{}, error) {
+	g.add(http.MethodGet, "/hello2", func(_ *Context) (interface{}, error) {
 		return helloWorld, nil
 	})
 
-	g.PUT("/hello", func(c *Context) (interface{}, error) {
+	g.PUT("/hello", func(_ *Context) (interface{}, error) {
 		return helloWorld, nil
 	})
 
-	g.POST("/hello", func(c *Context) (interface{}, error) {
+	g.POST("/hello", func(_ *Context) (interface{}, error) {
```

Comment:
these can work even if you don't give an underscore i.e
func(*Context) like this